README.md

```diff
@@ -4,8 +4,12 @@
 
 This is a plugin for TinyMCE 5 that allows users to specify what language their text is written in. The plugin wraps
  the desired text in `span` tags with a `lang` attribute for the specified language. Unspecified text is assumed to be
-  written in the page's language. This helps the resulting text comply with WCAG 2.0 3.1.2 Language of Parts: "The
-   human language of each passage or phrase in the content can be programmatically determined..."
+ written in the page's language. This helps the resulting text comply with WCAG 2.0 3.1.2 Language of Parts: "The
+ human language of each passage or phrase in the content can be programmatically determined..."
+   
+The plugin is intended for use where a few words of a different language are embedded in a passage of some other
+ language. It also works when you need to change the language of the entire passage, but not as well. For best results,
+ don’t select any text and just change in and out of different language modes while typing.
```

Comment:
I think rather than saying this, it would be better to explain how to select text. Like saying it works on words, sentences, and a single paragraph is good. It's limitation is just the multi-paragraph case, right?

Reply:
It also can't handle cases where your selection crosses over multiple existing languages.

Reply:
Maybe call out that limitation then? I just think users would potentially like to select a full paragraph and switch the language. Just want to make sure they know that is possible, albeit with some limitations.